http/test/node/mock-http-source.ts

```diff
@@ -0,0 +1,21 @@
+import 'mocha';
+import * as assert from 'assert';
+import { mockHTTPSource } from '../../lib/index';
```

Comment:
We probably want to change this to import from `../../src/index`, it should be okay since mocha is configured to compile any typescript files it finds as it goes.